libcontainer/console_linux.go

```diff
@@ -5,18 +5,23 @@ import (
 	"os"
 	"syscall"
 	"unsafe"
+
+	"github.com/docker/docker/pkg/term"
```

Comment:
Yeh we wondered about this but thought we saw it elsewhere (maybe some legacy that was needing to be removed, or just a mistake). Have just done a little copying and refactoring in the updated PR.